features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -563,6 +568,34 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                             />
                         )
                     }
+                    {
+                        claim && !READONLY_CLAIM_CONFIGS.includes(claim?.claimURI)
+                            && !hideSpecialClaims
+                            && UIConfig?.isClaimUniquenessValidationEnabled && (
```

Comment:
Change the condition order

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/3088d327c477257341387a57516db4d9929fbfaa